### PR TITLE
8.0 stock route transit

### PR DESCRIPTION
**Pull request by gurneyalex**

### CODE REVIEW

File: stock_route_transit/test/stock_users.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    password: sam
+    email: stockmanager@yourcompany.com
+-
+  I added groups for Stock Manager.
+-
+  !record {model: res.users, id: res_users_stock_manager}:
+    groups_id:
+      - stock.group_stock_manager
+-
+   Create a user as 'Stock User'
```
Comment (lepistone) [inline stock_route_transit/test/stock_users.yml]: this is not used.

File: stock_route_transit/__init__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Author: Alexandre Fayollle
```
Comment (lepistone) [inline stock_route_transit/__init__.py]: lll

File: stock_route_transit/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Alexandre Fayollle
```
Comment (lepistone) [inline stock_route_transit/__openerp__.py]: lll

File: stock_route_transit/model/__init__.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Author: Alexandre Fayollle
```
Comment (lepistone) [inline stock_route_transit/model/__init__.py]: lll

File: stock_route_transit/README.rst [hunk 5]
```diff
@@ -0,0 +1,11 @@
+Stock Route Transit
+===================
+
+
+This addon adds Transit route configuration to warehouses.
```
Comment (lepistone) [inline stock_route_transit/README.rst]: I'd love more explanations here.

**Comment by lepistone** [comment]
Thanks @gurneyalex ! Globally looks good! :+1: . I fixed a minor lint issue myself.

Before merging that, I'd:
- expand the readme, at least to say what this module is supposed to do, and if the same result can be archieved with manual configuration of routes (or not)
- The test is long and it's not clear to me where we want to go. Even with this style, maybe some descriptions can be expanded where it makes sense to understand what the module does (i.e. `check the routes` -> `check that a special route "transit" has been generated` or something like that)
- A good part of the module seems to be tested, but maybe we can remove for now the missing test on the dropshipping case (not sure on this one). If you think it doesn't work yet, we can just note that in the readme.

Thanks!

**Comment by lepistone** [comment]
Thanks for the improvements! It's :+1: for me.

**Comment by jgrandguillaume** [comment]
Hi,

Functionnal test + code riview. LGTM :+1:  when travis green.

Thanks for this.

Joël

**Comment by yvaucher** [comment]
:+1:

**Pull request merged**